db/schema.rb

```diff
@@ -24,6 +24,49 @@
     t.index ["user_id"], name: "index_changes_on_user_id", using: :btree
   end
 
+  create_table "countries", force: :cascade do |t|
```

Comment:
These changes shouldn't be here. There was a migration to remove these tables a while ago. Might be worth dumping your local DB and run `rake db:migrate` again